lib/modules/manager/gradle/parser.spec.ts

```diff
@@ -59,6 +59,125 @@ describe('modules/manager/gradle/parser', () => {
       });
     });
 
+    describe('Groovy: multi var assignments', () => {
+      it('simple map', () => {
+        const input = `
```

Comment:
```suggestion
        const input = codeBlock`
```

@zharinov shouldn't we also use `codeBlock` template helper?

Reply:
Yes, we do 👍